fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -248,6 +250,18 @@ public void setName(String newName) {
         name = newName;
     }
 
+    public void setQualifiedDbName(String qualifiedDbName) {
```

Comment:
this method should change to package privilege, prevent modification by other modules